packages/layout/src/tree-layout/layout-definition/definition-manager/index.js

```diff
@@ -0,0 +1,48 @@
+import DefinitionModel from './definitionModel';
+import { prepareTargetComponentMap, getComponent, placeComponent, placeGridComponent } from '../helper';
+
+export default class DefinitionManager {
+    constructor (componentMap, sequence, totalHeight, totalWidth) {
+        this._componentMap = componentMap;
+        this._prioritySequence = sequence;
+        this._totalHeight = totalHeight;
+        this._totalWidth = totalWidth;
+        this._targetComponentMap = null;
+    }
+
+    // create the config model
+    generateConfigModel () {
+        prepareTargetComponentMap(this);
+        const canvasComponent = this._targetComponentMap.get('canvas');
+        const definitionModel = new DefinitionModel();
+        let tempDefModel = definitionModel;
+        definitionModel.remainingHeight(this._totalHeight);
```

Comment:
Use getters to reference variables.

Reply:
All "let" variables must be declared at the beginning of a function.
Provide a line space after every set of declaration(after every const and let statements)

Reply:
variables of same class does's require a getter. Adds one more function call in stack